woocommerce/woocommerce-ios
# [Mac Catalyst] 1. Lay the groundwork: Check the box, make CocoaPods fixes, and resolve signing issues

Merge after #5653
Do not merge until #5662 and predecessors (#5661, #5659, #5657) are merged in.

### Description
This is the first in a series of changes which enable support for Mac Catalyst in the Woo app. This was a HACK week project, and is definitely not ready for production, so the last PR in the chain unchecks the Catalyst checkbox again, but once that's merged, working on Mac features will then only require checking the box on your branch.

 I'm keen that we merge these changes back down to develop even though it's going to be turned off, because otherwise the Catalyst support will go stale before anyone can use it.


### Testing instructions
Catalyst is not testable on this PR as there are still build errors at this stage. To try to build, you'd select `My Mac` as the target (not "Designed for iPad") and ensure that your Mac has been added to the development provisioning profile... but it will just error anyway.
Smoke testing iOS shows no changes to the app.


---
